### PR TITLE
13 create a schema reflecting the erd

### PR DESCRIPTION
**Pull request by polarbear23**

### CODE REVIEW

**Review by molloyconor511 — APPROVED** [review]

File: prisma/schema.prisma [hunk 1]
```diff
@@ -6,6 +6,105 @@ generator client {
...
+model Participant {
+  id         Int         @id @default(autoincrement())
+  createdAt  DateTime    @default(now())
+  updatedAt  DateTime    @updatedAt
+  Season     Season      @relation(fields: [seasonId], references: [id])
+  seasonId   Int
+  Team       Team?       @relation(fields: [teamId], references: [id])
+  teamId     Int?
+  placements Placement[]
+}
```
Comment (nathanSamosa) [inline prisma/schema.prisma:78]: Some are capitalized and some aren't, is there a purpose for these differences?

File: prisma/schema.prisma [hunk 2]
```diff
@@ -6,6 +6,105 @@ generator client {
...
+  rounds           Round[]
+  positionMappings PositionMapping[]
+}
+
+model Team {
+  id           Int           @id @default(autoincrement())
+  name         String
+  createdAt    DateTime      @default(now())
+  updatedAt    DateTime      @updatedAt
+  Season       Season        @relation(fields: [seasonId], references: [id])
```
Comment (bravint) [inline prisma/schema.prisma:64]: Nice catch, we didn't have it in the ERD

File: prisma/schema.prisma (hunk 1 above)
Comment (polarbear23) [reply prisma/schema.prisma:78]: good spot I will change it so that they are all lowercase for uniformity

File: prisma/schema.prisma [hunk 3]
```diff
@@ -6,6 +6,105 @@ generator client {
...
+  nationality        String?
+  email              String
+  competitorImageUrl String?
+  createdAt          DateTime    @default(now())
+  updatedAt          DateTime    @updatedAt
+  Competition        Competition @relation(fields: [competitionId], references: [id])
+  competitionId      Int
+}
+
+model Season {
```
Comment (bravint) [inline prisma/schema.prisma:47]: Needs entry for title (str)

**Review by bravint — CHANGES_REQUESTED** [review]
nice work, just those small changes needed, @nathanSamosa has picked up on inconsistencies in the way you name your relations, should be like you have it in line 32.

File: prisma/schema.prisma (hunk 3 above)
Comment (polarbear23) [reply prisma/schema.prisma:47]: will add it now

**Review by bravint — APPROVED** [review]

**Review by glowkeeper — APPROVED** [review]
Approved

**Pull request merged**